### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by makubi**
Update docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,17 +79,18 @@ protected void onActivityResult(final int requestCode, final int resultCode, fin
...
 #### Place a file into Belvedere’s internal storage
 Moreover, it’s possible to put your own data into Belvedere’s cache. To get access to an internal file, call:
 
 ```
 BelvedereResult file = belvedere.getFileRepresentation(“file_name.tmp”);
 ```
-Again, you’ll get a file object and an Uri. For example, you can use the file to open a FileOutputStream.
-Open or share an internal file
-Files that are available through Belvedere could be opened or shared with other apps. To do that, use the Uri you get from a `BelvedereResult`.
+Again, you’ll get a file object and an `Uri`. For example, you can use the file to open a `FileOutputStream`.
```
Comment (baz8080) [inline README.md]: It's an old typo, but please change `an Uri` to `a Uri`

Comment (makubi) [inline README.md]: Sure, done.

**Comment by baz8080** [comment]
@schlan good to merge?

**Comment by schlan** [comment]
yes, thanks to Mr. Kub 🐔

**Comment by makubi** [comment]
:chicken:

**Pull request merged**